coapcloudconf.raml

```diff
@@ -48,8 +47,8 @@ traits:
               {
                 "rt": ["oic.r.coapcloudconf"],
                 "at": "0f3d9f7fe5491d54077d",
```

Comment:
Need to add "if" here since baseline should return ALL Properties.

Reply:
Updated in PR #5 (Now Merged).